docs/npm.md

```diff
@@ -0,0 +1,45 @@
+# Microsoft Azure Container Networking
+
+## Azure Network Policy Manager
+
+`azure-npm` Network Policy plugin implements the [Kubernetes Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+
+The plugin is available on Linux platform. Windows support is planned.
+
+Azure-NPM serves as a distributed firewall for the Kubernetes cluster, and it can be easily controlled by `kubectl`.
+
+## Install
+
+Simply run
```

Comment:
Running the command below will bring up one azure-npm instance on each Kubernetes node.